src/dmd/mtype.d

```diff
@@ -1126,6 +1126,14 @@ extern (C++) abstract class Type : RootObject
         return mod == 0;
     }
 
+    /**
+     * Indicates wether the type is usable with `extern(c++)`.
+     */
+    bool isExternCppCompatible() const nothrow pure @nogc @safe
+    {
+        return true;
```

Comment:
This could simply be `final` and check `TY`

Reply:
okay done